src/kernels/Conv_Winograd_v30_3_1_gfx94x_fp32_f2x3_dilation2.inc

```diff
@@ -24,50 +24,77 @@
  *
  *******************************************************************************/
 
-.macro _buffer_store_dword vdata, vidx, srd, idx, mode
+.macro _buffer_store_short vdata, vidx, srd, idx, mode
 .if FORCE_CACHE_BYPASS_ON_STORE
-    buffer_store_dword \vdata, \vidx, \srd, \idx \mode sc0 sc1
+    buffer_store_short \vdata, \vidx, \srd, \idx \mode sc0 sc1
 .else
-    buffer_store_dword \vdata, \vidx, \srd, \idx \mode
+    buffer_store_short \vdata, \vidx, \srd, \idx \mode
 .endif
 .endm
```

Comment:
This macro should be reverted to `_buffer_store_dword` for fp32 kernels and should substitute `buffer_store_dword` instructions.

Reply:
done